src/components/rockets/Lists.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect } from "react"
+import { reserveRocket } from "../../redux/rockets/RocketsSlice"
+import { FetchData } from "../../redux/rockets/API"
+import {useDispatch, useSelector} from 'react-redux'
+import styles from "../../styles/rockets/rocket.module.css";
+
+export const Lists = ()=>{
+    const {rockets, status, error} = useSelector((state)=> state.rockets)
+    const Dispatch = useDispatch()
+    useEffect(()=>{
+        Dispatch(FetchData())
+    }, [Dispatch])
```

Comment:
- [ ] Kindly make sure the fetched data are not being added to the store whenever a user changes view / uses navigations.

    - ### APP BEHAVIOUR
    
   ![chrome-capture-2023-4-18 (2)](https://github.com/AbbasSarwar/SpaceTravelersHub/assets/87197412/7b921841-b652-434d-88f5-5674bb0d9b0d)

  

    _**Hint: Make sure to dispatch the `FetchedData()` action only when the `rockets`  array in the redux store is empty**_